Package.resolved

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
@Pinpickle what is this file? It looks like it's been generated but not filtered out by the gitignore...

Reply:
Whoops I thought I deleted this. I think it's a Swift 4 thing that I accidentally created while debugging.